compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/input/pointer/PointerIcon.js.kt

```diff
@@ -20,7 +20,40 @@ object DummyPointerIcon : PointerIcon
 
 internal data class BrowserCursor(val id: String): PointerIcon
 
+// Extracted from: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor
+// Note: cursor existence may vary in different browsers and OS's
+
 internal actual val pointerIconDefault: PointerIcon = BrowserCursor("default")
 internal actual val pointerIconCrosshair: PointerIcon = BrowserCursor("crosshair")
 internal actual val pointerIconText: PointerIcon = BrowserCursor("text")
-internal actual val pointerIconHand: PointerIcon = BrowserCursor("pointer")
\ No newline at end of file
+internal actual val pointerIconHand: PointerIcon = BrowserCursor("pointer")
+
+val ContextMenu: PointerIcon = BrowserCursor("context-menu")
```

Comment:
I think it's better to expose a way to create it from id instead of introducing many public variables.

@eymar @Schahen wdyt?

Reply:
Yeah the best approach would be to make BrowserCursor dataclass public or adding a function that returns a pointer icon browser cursor by passing an id string parameter

Reply:
i'll wait for @eymar for feedback before commiting the code in GetBrowserById

Reply:
i'll wait for @eymar for feedback before commiting the code in GetBrowserById

Reply:
Yeah, i thought about making it an extension but was not sure. Looks good to me